prensa_latas/src/components/QrCodeConfirmation/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+// import { Link } from 'react-router-dom';
+import style from './style';
+import logo from '../../assets/images/logo.svg';
+import check from '../../assets/images/check_circle.svg';
+
+class QrCodeConfimartion extends Component {
+    render () {
+        return (
+            <div style={style.qrcodeConfirmation}>
+                <div style={style.connected}>
+                    <p>Conectado à máquina</p>
+                </div>
+                <p style={style.machine}>Nome da Máquina</p>
```

Comment:
The machine's name should be rendered by a `props`, something like: `this.props.name`. It will allow using this component at any case.